packages/arb-token-bridge-ui/src/util/infura.ts

```diff
@@ -0,0 +1,99 @@
+import { providers } from 'ethers'
+import { Chain, ChainProviderFn } from 'wagmi'
+
+import { ChainId, rpcURLs } from './networks'
+
+export function infuraProvider<TChain extends Chain>(): ChainProviderFn<
```

Comment:
let's rename to `customInfuraProvider` to not conflict with wagmi's
let's also add a comment with a link to the reference implementation (https://github.com/wevm/wagmi/blob/wagmi%400.12.13/packages/core/src/providers/infura.ts)